packages/core-types/tests/integration/components/field-editors/integer-editor-test.js

```diff
@@ -0,0 +1,57 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, triggerEvent, waitFor } from '@ember/test-helpers';
+import Object from '@ember/object';
```

Comment:
this overrides the `Object` global. it's better to `import EmberObject from '@ember/object';`. I think ESLint should also have a rule for this, but maybe it's not enabled? 🤔

Reply:
I kind of felt it wasn't the thing I wanted :)